### PR TITLE
adding schedule to istio twich live stream page

### PR DESCRIPTION
**Pull request by lidderupk**
@nibalizer Please review. Thank you.

### CODE REVIEW

File: content/blog/2018/istio-twitch-stream/index.md [hunk 1]
```diff
@@ -21,6 +21,30 @@ The stream is going to be a full day of Istio content. Hopefully we'll have a go
 
 It's easy! Just navigate [here](https://twitch.tv/ibmcode) on August 17th starting at 10 AM Pacific.
 
+## Schedule
+
+All times are `PDT`.
+
+| Time | Speaker | Affiliation |
+| --- | --- | --- |
+| 10:00 - 10:30 | `Spencer + Lisa` | `IBM / Portworx` |
```
Comment (sdake) [inline content/blog/2018/istio-twitch-stream/index.md]: i'd recommend same syntax throughout this change. e.g. `Spencer Krum / Lisa (don't know last name:)

File: content/blog/2018/istio-twitch-stream/index.md [hunk 2]
```diff
@@ -21,6 +21,30 @@ The stream is going to be a full day of Istio content. Hopefully we'll have a go
...
+| 10:30 - 11:00 | `Lin Sun / Spencer Krum / Sven` | `IBM / Google` |
```
Comment (sdake) [inline content/blog/2018/istio-twitch-stream/index.md]: Add Sven's last name please

File: content/blog/2018/istio-twitch-stream/index.md [hunk 3]
```diff
@@ -21,6 +21,30 @@ The stream is going to be a full day of Istio content. Hopefully we'll have a go
...
+| 11:00 - 11:10 | `Lin / Spencer` | `IBM` |
```
Comment (sdake) [inline content/blog/2018/istio-twitch-stream/index.md]: last names

Comment (lidderupk) [reply content/blog/2018/istio-twitch-stream/index.md]: Added. Thanks @sdake !

**Comment by sdake** [comment]
If you want this PR to appear on the istio.io website, you will need to cherrypick to the release-1.0 branch.

Cheers
-steve

**Comment by lidderupk** [comment]
@sdake, not sure if I have permissions to do so.

**Pull request merged**